qiskit/ignis/experiments/calibration/instruction_data/parameter_table.py

```diff
@@ -0,0 +1,250 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Local database components.
+
+# TODO add detailed description of databases.
+"""
+
+from typing import Dict, Union, Iterable, Optional, List, Tuple
+
+import pandas as pd
+
+from qiskit import circuit
+from qiskit.ignis.experiments.calibration import types
+from qiskit.ignis.experiments.calibration.exceptions import CalExpError
+from qiskit.ignis.experiments.calibration.instruction_data import utils
+
+
+class PulseParameterTable:
+    """A database to store parameters of pulses.
+
+    Each entry of this database represents a single parameter associated with the specific pulse,
+    and the pulse template is stored in another relational database.
+
+    You can search for the specific entry by filtering or you can directly generate
+    keyword argument for the target pulse factory.
+    """
+    TABLE_COLS = ['qubits', 'channel', 'pulse_name', 'series', 'gate_id',
```

Comment:
Replace `series` with `context` (or `scope`) and adjust accordingly in the rest of the code?

Reply:
I think `gate_id` is kind of scope. `series` can be separate parameter table -- actually we don't need to have two series in the same parameter table. We may be able to define dedicated parameter table for each series. Something like
```
inst_def1 = provider.get_calibration(series='my_cal1')
inst_def2 = provider.get_calibration(series='my_cal2')
```
I'm not sure if we can do this.

Reply:
That could be an option.